docs/concepts/browsers-used-by-office-web-add-ins.md

```diff
@@ -54,6 +54,9 @@ Setting breakpoints in the [Microsoft Edge DevTools](https://www.microsoft.com/p
 
 One known cause is that Microsoft Edge requires that localhost be given a loopback exemption on the development computer. Follow the instructions at [Cannot open add-in from localhost](/office/troubleshoot/error-messages/cannot-open-add-in-from-localhost).
 
+### Get errors trying to download a PDF file
+
+Directly downloading blobs as PDF files in an add-in is not supported when Edge is the browser. The workaround is to create a simple web application that downloads blobs as PDF files. In your add-in call the `Office.context.ui.openBrowserWindow(url)` method and pass the URL of the web application. This will open the web application in a browser window outside of Office.
```

Comment:
Do we publicly use "workaround", I feel like it also may not translate? Maybe "we suggest creating..." Otherwise, seems good!

Reply:
Yes, we use "workaround" in several places in our docs.